linode/manager
# Cancel Request POC

### Purpose

Implement Axios' cancellable token to prevent the constant need for...

```
if (!this.mounted) return;
```

### To test

Just switch to the `/domains` tab and then switch between it and `/managed` as fast as possible. And then observe how there's no memory leak console warning

And in the network tab, you should see the requests being highlighted red and marked `canceled`
